packages/components/src/Components/FormElements/InputText/InputText.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+import { colors, formColors } from '../../../Utils/colors';
+
+const styles = {
+  text: css`
+    border: 1px solid ${formColors.colorInputBorder};
+    background: ${formColors.colorInputBg};
+    color: ${formColors.colorInputText};
+    border-radius: 2px;
+    padding: 1em;
+    font-size: 16px;
+    box-shadow: inset 0 0 0 1px transparent;
+    box-sizing: border-box;
+    &:placeholder {
+      color: ${formColors.colorInputPlaceholder};
+    }
+    &:hover {
+      border-color: ${formColors.colorInputFg};
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputFg};
+    }
+    &:focus {
+      border-color: ${formColors.colorInputBorderFocus};
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputBorderFocus};
+      outline: none;
+    }
+  `,
+  textError: css`
+    border-color: ${formColors.colorInputBorderError};
+    box-shadow: inset 0 0 0 1px ${formColors.colorInputBorderError};
+  `,
+  textDisabled: css`
+    color: ${formColors.colorInputBorderError};
+    background-color: ${formColors.colorInputBgDisabled};
+    &:hover {
+      border-color: ${formColors.colorInputBorder};
+      box-shadow: inset 0 0 0 1px transparent;
+    }
+  `,
+  label: css`
+    color: ${colors.text};
+    padding-bottom: 5px;
+  `,
+  labelRequired: css`
+    :after {
+      content: ' *';
+      color: ${colors.maximumRed};
+    }
+  `,
+  labelWithError: css`
+    color: ${colors.maximumRed};
+  `,
+  errorText: css`
+    color: ${colors.maximumRed};
+    padding-top: 5px;
+  `,
+};
+
+/**
+ * Renders an Input type text.
+ *
+ * Example:
+ *
+ * @code
+ * <InputText required />
+ * @endcode
+ */
+const InputText = props => {
+  const {
+    name,
+    error,
+    errorText,
+    disabled,
+    required,
+    placeholder,
+    htmlAttributes,
+  } = props;
+  const labelClasses = [styles.label];
+  const cbClasses = [styles.text];
+  const etClasses = [styles.errorText];
+  if (required) {
+    labelClasses.push(styles.labelRequired);
+  }
+  if (error) {
+    labelClasses.push(styles.labelWithError);
+    cbClasses.push(styles.textError);
+  }
+  if (disabled) {
+    cbClasses.push(styles.textDisabled);
+  }
+
+  return (
+    <label id={`id-${name}-label`} htmlFor={`id-${name}`}>
```

Comment:
I wonder whether the textfield should have a label included by itself? Wouldn't you wrap the input component in the label component instead? What are your thoughts?

Reply:
@koskinpark Did you see this comment?

Reply:
@dawehner Yes sure, but right now i don't have free time. I'l try to update this PR next week.

Reply:
@koskinpark No worries, let me know when you had time to work on it again :)

Reply:
@dawehner please can you review and tell me if it's ok/nok for you